portfolio/src/main/webapp/index.html

```diff
@@ -27,4 +27,7 @@
           <div class="items items2 items3"> <a href="#footer">Contact</a></div>    
         </div>
         </div>
-      </nav>
\ No newline at end of file
+      </nav>
+      </section>
```

Comment:
This should be moved to the left one indentation level. Can be fixed in a future change.